modules/nf-core/metabat2/metabat2/main.nf

```diff
@@ -35,14 +35,10 @@ process METABAT2_METABAT2 {
         $depth_file \\
         -t $task.cpus \\
         --saveCls \\
-        -o metabat2/${prefix}
+        -o ${prefix}
```

Comment:
anything in the modules.config that needs to be adapted to keep the expected publishDir structure?